File: system-test/java/src/test/java/com/optivem/eshop/systemtest/legacy/mod05/smoke/external/ErpSmokeTest.java
package com.optivem.eshop.systemtest.legacy.mod05.smoke.external;

import com.optivem.eshop.systemtest.legacy.mod05.base.BaseDriverTest;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static com.optivem.eshop.dsl.common.ResultAssert.assertThatResult;

class ErpSmokeTest extends BaseDriverTest {
    @BeforeEach
    void setUp() {
        setUpExternalDrivers();
    }

    @Test
    void shouldBeAbleToGoToErp() {
        var result = erpDriver.goToErp();
        assertThatResult(result).isSuccess();
    }
}

File: system-test/dotnet/SystemTests/Legacy/Mod05/SmokeTests/External/ErpSmokeTest.cs
using Common;
using SystemTests.Legacy.Mod05.Base;
using Shouldly;
using Xunit;

namespace SystemTests.Legacy.Mod05.SmokeTests.External;

public class ErpSmokeTest : BaseDriverTest
{
    public override async Task InitializeAsync()
    {
        await base.InitializeAsync();
        SetUpExternalDrivers();
    }

    [Fact]
    public async Task ShouldBeAbleToGoToErp()
    {
        var result = await _erpDriver!.GoToErpAsync();
        result.ShouldBeSuccess();
    }
}










